contrib/metrics-exporter/Dockerfile

```diff
@@ -0,0 +1,24 @@
+
+FROM golang:1.19 as builder
```

Comment:
Can we add this to dependencies.yaml:

https://github.com/cri-o/cri-o/blob/6e861a881c1c7124836228c7c7b9abb696f6c283/dependencies.yaml#L1-L14

Reply:
Done!